kivy_ios/recipes/itsdangerous/__init__.py

```diff
@@ -14,11 +14,10 @@ def install(self):
         build_dir = self.get_build_dir(arch.arch)
         hostpython = sh.Command(self.ctx.hostpython)
         build_env = arch.get_env()
-        dest_dir = join(self.ctx.dist_dir, "root", "python")
+        dest_dir = join(self.ctx.dist_dir, "root", "python3")
         build_env['PYTHONPATH'] = join(dest_dir, 'lib', 'python3.8', 'site-packages')
         cmd = sh.Command("sed")
```

Comment:
I wish the linter could post that comment on the exact line for us
```
kivy_ios/recipes/itsdangerous/__init__.py:19:9: F841 local variable 'cmd' is assigned to but never used
```

Reply:
Would be nice. But, I should just tun the pep8 checker locally I suppose... Not in that habit on this repo yet. Should add it as a hook...

Reply:
I also often don't run it locally because we think changes are trivial, but it's easy to forget something or make typos. That's why we have CI to gently remind us :D